deploy/cephcsi/image/Dockerfile

```diff
@@ -67,6 +67,10 @@ LABEL maintainers="Ceph-CSI Authors" \
 
 COPY --from=builder ${SRC_DIR}/_output/cephcsi /usr/local/bin/cephcsi
 
+RUN dnf -y update \
```

Comment:
This is downloading the updates (and metadata) 2x. Could this be structured a little different so that it is more efficient? I am thinking of something like this (simplified):

```Dockerfile
# update the base image
FROM ${BASE_IMAGE} as base_updated

RUN dnf -y update \
    && ...

# builder contains extra development tools
FROM base_updated AS builder

RUN make cephcsi

# final image for deployments
FROM base_updated

COPY --from=builder ...
```

Reply:
The metadata would always be downloaded twice if it is cleaned after use to keep layers "small". Will try moving updates around.